.github/workflows/deploy.yml

```diff
@@ -1,7 +1,7 @@
 name: Deploy VitePress site to Pages
 
 on:
-  # Runs on pushes targeting the `develop` branch.
+  # Runs on pushes targeting the `main` branch.
```

Comment:
This has been changed, but not the line 6 below (the actual rule)

Reply:
Yes, will change before merging to `main`